doc/content/getting-started/migrating-from-networks/device-json.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Create Device.json"
+description: ""
+weight: 10
+---
+
+{{% tts %}} allows you to import devices from other networks using a JSON file describing those devices. Devices imoported this way can be migrated without the need for a rejoin.
+
+## Required Fields in `devices.json`
+
+- [device_id]({{< ref "reference/glossary#device-id" >}}) 
```

Comment:
Perhaps this list could be a table instead? cc @benolayinka

Reply:
Good idea.